aws/inputseeders/aws/masterdata.json

```diff
@@ -952,6 +1048,176 @@
                     }
                 }
             }
+        },
+        "_awswin" : {
+            "Modes" : {
+                "*" : {
+                    "ecs" : {
+                        "Profiles" : {
+                            "Storage" : "ecs_awswin",
+                            "LogFile" : "awswinlog"
+                        },
+                        "AutoScaling": {
+                            "StartupTimeout": "20M"
+                        },
```

Comment:
20 minutes for each box? seems quite long.

Reply:
The doco lists up to 15 minutes for a box to start and 15 minutes to join a cluster. If I've read the doco aright, that's a maximum time for a response (not a minimum), correct?

Reply:
Its more the impact on a system if the box fails to start correctly. 20 minutes to wait for an instance to timeout when it didn't start properly is a pretty big wait during scaling activities.

Reply:
If an ECS has a data volume it needs to do an NTFS format plus doco warns up to 15 minutes to join the cluster and can be successful

I couldn't locate how long nix was expected to take to join a cluster, but given the above it didn't seem too much to double it to 20M. Thoughts?

The risk point is an ECS that doesn't positively succeed nor positively fail but doesn't start correctly, correct?

Reply:
Ok sounds reasonable then. Just wanted to make sure that some of the dev work wasn't bleeding into the default setup.